File: 6th Semester/4 programs 4 languages assignment/ex3.py
# Desenvolva um programa que leia a idade de uma pessoa e informe a sua classe eleitoral:
# ➢ não eleitor (abaixo de 16 anos);
# ➢ eleitor obrigatório (entre a faixa de 18 e menor de 65 anos);
# ➢ eleitor facultativo (de 16 até 18 anos e maior de 65 anos, inclusive).

idade = int(input('Insira a idade: '))

if idade < 16:
    print('Sem idade para votar')
elif idade < 18 or idade >= 65:
    print('Eleitor facultativo')
else:
    print('Eleitor obrigatório.')

File: 6th Semester/Programming Paradigms/NP1 Exercises - 4 programs 4 languages assignment/ex3.cs
using System;
  
public class ex3 {

    static public int leitorInt(string message) {
        int numberInt;
        while (true)
        {
            Console.Write(message);
            if (int.TryParse(Console.ReadLine(), out numberInt)) return numberInt;
            Console.WriteLine("Valor inválido. Tente novamente.");
        }
    }
  
    static public void Main()
    {

        int idade = leitorInt("Insira a idade: ");
        if (idade < 16) Console.WriteLine("Sem idade para votar");
        else if (idade < 18 || idade >=65) Console.WriteLine("Eleitor facultativo");
        else Console.WriteLine("Eleitor obrigatório");
    }
}